lib/json/test/valid/string_in_object.json

```diff
@@ -1 +1,3 @@
-{"key":"hello this is a string"}
\ No newline at end of file
+{
```

Comment:
Oh, I see. 
I didn't know that. Maybe you should mention it in credit section in the ReadMe file !
I'll drop the changes to the JSON files.